.github/workflows/docs.yaml

```diff
@@ -0,0 +1,50 @@
+name: Docs
+
+on:
+    push:
+        branches: [main]
+    release:
+        types: [released]
+
+jobs:
+    docs:
+        runs-on: ubuntu-latest
+
+        steps:
+        - uses: actions/checkout@v2
+
+        - name: Setup Python
+          uses: actions/setup-python@v2
+          with:
+              python-version: 3.8
+
+        - name: Install Dependencies
+          run: pip install .[doc]
+
+        - name: Build HTML artifact
+          run: "python release_docs.py"
```

Comment:
why is this in quotes?

Reply:
I copied it from the other sphinx CI action I was looking into. Removed the quotes.